Classes/Command/StyleguideCommandController.php

```diff
@@ -115,8 +117,15 @@ public function renderCommand($prototypeName, $packageKey = null, ?string $useCa
             'props' => $convertedProps,
             'locales' => $convertedLocales
         ]);
-
-        $this->output($fusionView->render());
+        $result = $fusionView->render();
+        if ($result instanceof ResponseInterface) {
+            return (string) $result->getBody();
+        }
+        if ($result instanceof StreamInterface) {
+            return (string) $result;
+        }
+        // support for Neos 8.3
+        $this->output();
```

Comment:
i believe this is wrong for 8.3 should be either `$this->output($result);` or `return $result` :D

Reply:
Yeh, totally looks wrong. Will take a look tomorrow

Reply:
Fixed with https://github.com/sitegeist/Sitegeist.Monocle/releases/tag/v7.10.6